src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,90 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
```

Comment:
You have used `@BeforeAll` to initialize your `RegistrationServiceImpl` class instance. This means that the same instance of `RegistrationServiceImpl` will be used for all test cases. This is not a problem in your case since `RegistrationServiceImpl` does not have any state, but be aware that using `@BeforeAll` can lead to problems if the class under test has state that is modified during the tests.